{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -37,12 +37,38 @@ style = "pep440"
 line-length = 100
 target-version = ["py38", "py39", "py310"]
 
-[tool.isort]
-profile = "black"
-multi_line_output = 3
-line_length = 100
-include_trailing_comma = true
-reverse_relative = true
+[tool.ruff]
+ extend-exclude = [
+     "tests/output",
+     "tests/**/output"
+ ]
```

Comment:
Not sure if this is necessary here. It's in the `linkml` repo's config because of its unique test setup that I wouldn't expect projects create from this cookiecutter to recreate.

Reply:
Thanks! Removed it and replaced it with 
```
extend-ignore = [
    "D211",  # `no-blank-line-before-class`
    "D213",  # `multi-line-summary-second-line`
    ]
```